src/Orleans/Orleans.csproj

```diff
@@ -74,8 +74,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Yes, this is the default NuGet behaviour, and it is E.V.I.L. 
It directly leads to a whole set of binding problems down the road!

Don't even get me started about how NuGet used to add fully versioned assembly name references and then also add SpecifiedVersion=False. Duh - logic failure! ;)

Short assembly names do exactly what you want here and reduce-friction in life. 

Just keep politely saying "No means No" every time NuGet does this to you :)